src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Serialization/ModelDeserializer.cs

```diff
@@ -92,28 +93,69 @@ private async Task<object> DeserializeXmlAsync(Stream stream)
         {
             Error = null;
 
+            string streamContent = null;
             try
             {
+                // In this first try block we assume that the namespace is the same in the model
+                // and in the XML. This includes no namespace in both.
                 using StreamReader reader = new StreamReader(stream, Encoding.UTF8);
-                string content = await reader.ReadToEndAsync();
+                streamContent = await reader.ReadToEndAsync();
 
-                using TextReader sr = new StringReader(content);
+                _logger.LogInformation(streamContent);
```

Comment:
Should this be "downgraded" to a debug or trace log?

Reply:
Oh, this must be removed I think.